APIv1.yaml

```diff
@@ -0,0 +1,490 @@
+                    required: true
+                    example: 5
+                    schema:
+                        type: integer
+            responses:
+                '201':
+                    description: Share node object
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Share"
+                '404':
+                    description: Not found
+        delete:
+            summary: Unshare a table
+            description: Shipped with v0.4.0.
+            tags:
+                - Shareing
+                - Tables
+            operationId: deleteTableShare
+            security:
+                -   basicAuth: [ ]
+            parameters:
+                -   name: shareId
+                    in: path
+                    description: ID for the share
+                    required: true
+                    example: 5
+                    schema:
+                        type: integer
+            responses:
+                '201':
+                    description: Deleted share node object
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Share"
+        put:
+            summary: Update permission
+            description: Shipped with v0.4.0.
+            tags:
+                - Shareing
+            operationId: updateSharePermission
+            security:
+                -   basicAuth: [ ]
+            parameters:
+                -   name: shareId
+                    in: path
+                    description: Share ID
+                    required: true
+                    example: 5
+                    schema:
+                        type: integer
+                -   name: permissionType
+                    in: query
+                    description: Permission type (read, create, update, delete, manage)
+                    required: true
+                    schema:
+                        type: string
+                -   name: permissionValue
+                    in: query
+                    description: Value for the permission type
+                    required: true
+                    schema:
+                        type: boolean
+            responses:
+                '201':
+                    description: Share node object
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Share"
+
+    /table/{tableId}/rows:
+        get:
+            summary: Get an array of rows
+            description: The first row contains the column titles. There is a default row limit by 1000 rows. If you need more (or want to limit less), you can set a limit on your own. Additionally set an offset if needed, would be good for lazy loading etc. Shipped with v0.3.0.
```

Comment:
@datenangebot Are you sure this API is present since `v0.3.0` of the app? Calling this endpoint in `v0.3.3` of the app with a valid `tableId` in the path results in `HTTP 405 Method not allowed`.